ereolen_support/ereolen_material.form.inc

```diff
@@ -41,6 +41,17 @@ function ereolen_material_form($form, &$form_state) {
     '#required' => TRUE,
   ];
 
+  $form['ereolen_material_type'] = [
+    '#type' => 'select',
+    '#title' => t('Type'),
+    '#options' => [
+      t('E-Book') => t('E-Book'),
+      t('Audiobook') => t('Audiobook'),
+      t('E- and audiobook') => t('E- and audiobook'),
```

Comment:
I would write this as `E-book and audiobook` (I'm not sure English works the same way as Danish when it comes to using hyphens like this).